MANIFEST.in

```diff
@@ -1,8 +0,0 @@
-include requirements.txt
```

Comment:
in the old setup, manifest.in ensured that the license file, contributing.md docs and tests are included in the pypi package. How will this be done in poetry?

Reply:
Gotcha, I just updated the `include` field of pyproject.toml to include these assets and verified the zipped bundle.